contrib/spring_sleuth/README.md

```diff
@@ -0,0 +1,50 @@
+# OpenCensus Spring Sleuth
+[![Build Status][travis-image]][travis-url]
+[![Windows Build Status][appveyor-image]][appveyor-url]
+[![Maven Central][maven-image]][maven-url]
+
+The *OpenCensus Spring Sleuth for Java* is a library for automatically
+propagating the OpenCensus trace context when working with [Spring Sleuth][spring-sleuth-url].
+
+This is an __experimental component__.
```

Comment:
please add that this is therefore not supported. For example, the sleuth support channel is quite busy, we can't really afford take on support requests for things like this. We are already swamped supporting the versions of sleuth out there that are not hacked (or less formally hacked :P )

Another way is to say:

This is an __experimental component__, so please bring feedback to https://gitter.im/census-instrumentation/Lobby not the usual sleuth channel https://gitter.im/spring-cloud/spring-cloud-sleuth

Reply:
Absolutely.